modules/apps/27-interchain-accounts/controller/types/codec.go

```diff
@@ -8,4 +8,5 @@ import (
 // RegisterInterfaces registers the interchain accounts controller message types using the provided InterfaceRegistry
 func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 	registry.RegisterImplementations((*sdk.Msg)(nil), &MsgRegisterAccount{})
+	registry.RegisterImplementations((*sdk.Msg)(nil), &MsgSubmitTx{})
```

Comment:
I think we can consolidate these registrations

```suggestion
	registry.RegisterImplementations(
	    (*sdk.Msg)(nil), 
	    &MsgRegisterAccount{},
	    &MsgSubmitTx{},
	)
```